.github/workflows/ci.yml

```diff
@@ -19,41 +19,66 @@ jobs:
       uses: actions/setup-node@v3
       with:
         node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+        cache-dependency-path: 'packages/**/package-lock.json'
     - run: npm ci
     - run: npm run lint
     - run: npm run cs-check
 
     - name: Build with Netlify badge
-      if: github.ref != 'refs/heads/main'
+      if: github.ref != 'refs/heads/main' && matrix.node-version != '14.x'
       run: npm run build
       env:
         VITE_SHOW_NETLIFY_BADGE: true
         NODE_OPTIONS: --max_old_space_size=4096
+
     - name: Build
-      if: github.ref == 'refs/heads/main'
+      if: github.ref == 'refs/heads/main' && matrix.node-version != '14.x'
       run: npm run build
       env:
         NODE_OPTIONS: --max_old_space_size=4096
 
+    # Docusaurus doesn't support Node 14. These tasks can be removed once Node 14 is EOL (May 2023)
+    - name: Build with Netlify badge (no docs)
+      if: github.ref != 'refs/heads/main' && matrix.node-version == '14.x'
+      run: npx nx run-many --target=build --exclude=docs
+      env:
+        VITE_SHOW_NETLIFY_BADGE: true
+        NODE_OPTIONS: --max_old_space_size=4096
+    - name: Build (no docs)
+      if: github.ref == 'refs/heads/main' && matrix.node-version == '14.x'
+      run: npx nx run-many --target=build --exclude=docs
+      env:
+        NODE_OPTIONS: --max_old_space_size=4096
+
     - if: matrix.node-version == '18.x'
       uses: actions/upload-artifact@v3
       with:
-        name: dist
+        name: playground
         path: packages/playground/dist
+    - if: matrix.node-version == '18.x'
+      uses: actions/upload-artifact@v3
+      with:
+        name: docs
+        path: packages/docs/build
     - run: npm test
-
-  deploy_playground:
+  deploy_playground_and_docs:
     runs-on: ubuntu-latest
     if: github.ref == 'refs/heads/main'
-    name: Deploy playground to GitHub Pages
+    name: Deploy playground and docs to GitHub Pages
     needs: [build]
     steps:
-      - name: Download built files
+      - name: Download built playground
         uses: actions/download-artifact@v3
         with:
-          name: dist
+          name: playground
           path: dist
-      - name: Deploy playground
+      - name: Download built docs
+        uses: actions/download-artifact@v3
+        with:
+          name: docs
+          path: dist/docs
```

Comment:
Basically, download the docs artifact and stick it in the `docs` folder of the playground artifact. When we deploy, we'll end up with both sites, with the docs pages in the docs folder.